### PR TITLE
nautilus: cephfs-shell: Convert paths type from string to bytes

### PR DESCRIPTION
**Pull request by varshar16**
https://tracker.ceph.com/issues/41269

### CODE REVIEW

**Comment by varshar16** [comment]
jenkins retest this please

**Comment by yuriw** [comment]
[wip-yuri6-testing-2019-10-01-1605-nautilus](https://trello.com/c/5DZ906lA/701-wip-yuri6-testing-2019-10-01-1605-nautilus)

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -244,20 +232,22 @@ def copy_from_local(local_path, remote_path):
...
-            poutput("error: no permission to read local file %s" % local_path)
+            perror('error: no permission to read local file {}'.format(
+                   local_path.decode('utf-8')), end='\n', apply_style=True)
             return
         stdin = 1
     try:
         fd = cephfs.open(remote_path, 'w', 0o666)
     except libcephfs.Error:
-        poutput("error: no permission to write remote file %s" % remote_path)
+        perror('error: no permission to write remote file {}'.format(
```
Comment (smithfarm) [inline src/tools/cephfs/cephfs-shell]: this change from poutput -> perror is not part of the commit being cherry-picked :-(

This complicates the rebase of https://github.com/ceph/ceph/pull/29764
